internal/config/config_test.go

```diff
@@ -121,26 +121,15 @@ func TestConfig(t *testing.T) {
 		}
 
 		// Verify derived configs.
-		dynamoDB := config.DynamoDBConfig{
```

Comment:
without these derive values, what exactly are we verifying with require.Equal(expectedAWS, cfg.AWS)?

Reply:
there are a few other values like:
IsLocalStack:           true,
IsResetLocal:           true,
PresignedUrlExpiration: 30 * time.Minute,

Reply:
renamed the test and also moved entire aws stuff verification to its own test